docs/release_notes/ibexa_dxp_v5.0_deprecations.md

```diff
@@ -0,0 +1,575 @@
+| `ezurlalias_ml`                                         | `ibexa_url_alias_ml`                                                      |
+| `ezurlalias_ml_incr`                                    | `ibexa_url_alias_ml_incr`                                                 |
+| `ezurlwildcard`                                         | `ibexa_url_wildcard`                                                      |
+| `ezuser`                                                | `ibexa_user`                                                              |
+| `ezuser_accountkey`                                     | `ibexa_user_accountkey`                                                   |
+| `ezuser_role`                                           | `ibexa_user_role`                                                         |
+| `ezuser_setting`                                        | `ibexa_user_setting`                                                      |
+
+## Field type identifiers
+
+Several field type identifiers have changed.
+
+| Old identifier (`legacy_alias`) | New identifier (`alias`)        |
+|:--------------------------------|:--------------------------------|
+| `ezauthor`                        | `ibexa_author`                    |
+| `ezbinaryfile`                    | `ibexa_binaryfile`                |
+| `ezboolean`                       | `ibexa_boolean`                   |
+| `ezcontentquery`                  | `ibexa_content_query`             |
+| `ezcountry`                       | `ibexa_country`                   |
+| `ezdate`                          | `ibexa_date`                      |
+| `ezdatetime`                      | `ibexa_datetime`                  |
+| `ezemail`                         | `ibexa_email`                     |
+| `ezfloat`                         | `ibexa_float`                     |
+| `ezform`                          | `ibexa_form`                      |
+| `ezgmaplocation`                  | `ibexa_gmap_location`             |
+| `ezimage`                         | `ibexa_image`                     |
+| `ezimageasset`                    | `ibexa_image_asset`               |
+| `ezinteger`                       | `ibexa_integer`                   |
+| `ezisbn`                          | `ibexa_isbn`                      |
+| `ezkeyword`                       | `ibexa_keyword`                   |
+| `ezlandingpage`                   | `ibexa_landing_page`              |
+| `ezmatrix`                        | `ibexa_matrix`                    |
+| `ibexa_measurement`               | `ibexa_measurement`               |
+| `ezmedia`                         | `ibexa_media`                     |
+| `ezobjectrelation`                | `ibexa_object_relation`           |
+| `ezobjectrelationlist`            | `ibexa_object_relation_list`      |
+| `ibexa_product_specification`     | `ibexa_product_specification`     |
```

Comment:
ibexa_measurement and ibexa_product_specification are the same before and after.